schema_editor/app/scripts/views/recordtype/detail-add-edit-partial.html

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Alright, let's just leave it for now. If time goes by and they don't diverge, and then there needs to be a structural rework for styling/etc., then we can combine them.